app/views/users/show.html.erb

```diff
@@ -1,13 +1,21 @@
-<div class="col-md-4 col-md-4 col-md-4 col-md-4">
-  <%if @user.image%>
-    <%= image_tag @user.image, alt: "Profile Pic", size: "150x200"%>
-  <%else%>
-    <%= image_tag "http://www.eurogeosurveys.org/wp-content/uploads/2014/02/default_profile_pic.jpg", alt: "Profile Pic", size: "150x200"%>
-  <%end%>  
-  <p><%= @user.name%></p>
-  <p>Date Joined: <%= @user.created_at.strftime('%c')%></p>
-  <p>Current City: <%= @user.current_city%></p>
+<div class="row">
+  <div class="col-md-4 col-md-4 col-md-4 col-md-4">
+    <%if @user.image%>
+      <%= image_tag @user.image, alt: "Profile Pic", size: "150x200"%>
+    <%else%>
+      <%= image_tag "http://www.eurogeosurveys.org/wp-content/uploads/2014/02/default_profile_pic.jpg", alt: "Profile Pic", size: "150x200"%>
```

Comment:
Ok, now we have this tricky default image logic in two places (see comment above). Definitely stop and refactor. -- Can you solve this problem outside of the view?